src/astro/sql/operators/sql_decorator.py

```diff
@@ -6,30 +6,27 @@
 You may obtain a copy of the License at
 
     http://www.apache.org/licenses/LICENSE-2.0
-
+from sqlalchemy import create_engine
```

Comment:
is this a typo?

Reply:
:joy: completely, a traditional `CTRL-C` + `CTRL-V` mistake